data/movements/scripts/tiles.lua

```diff
@@ -15,15 +15,15 @@ function onStepIn(creature, item, position, fromPosition)
 
 	if item.actionid >= 1000 then
 		if player:getLevel() < item.actionid - 1000 then
-			player:teleportTo(fromPosition, false)
 			position:sendMagicEffect(CONST_ME_MAGIC_BLUE)
 			player:sendTextMessage(MESSAGE_INFO_DESCR, "The tile seems to be protected against unwanted intruders.")
+			return false
 		end
 		return true
 	end
 
 	if Tile(position):hasFlag(TILESTATE_PROTECTIONZONE) then
-		local lookPosition = player:getPosition()
+		local lookPosition = position
```

Comment:
Do you really need to alias `position` to `lookPosition` or could you use `position` straight away?